src/UnityExtension/Assets/Editor/GitHub.Unity/EntryPoint.cs

```diff
@@ -53,7 +54,8 @@ private static void Initialize()
                     Logging.Error(ex, "Error rotating log files");
                 }
 
-                Debug.Log("Initialized GitHub for Unity log file: " + logPath);
+                logInfo += " and Unity log file: " + logPath;
+                Debug.Log(logInfo);
```

Comment:
There's really no need to have a separate variable to store the message before outputting, you're overcomplicating things here.

Reply:
Okay, I'll just redo that